packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -63,9 +63,9 @@ export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivEle
   isVertical?: boolean;
   /** Disables border bottom tab styling on tabs. Defaults to false. To remove the bottom border, set this prop to true. */
   hasNoBorderBottom?: boolean;
-  /** Aria-label for the left scroll button */
+  /** Aria-label for the back scroll button */
```

Comment:
can you deprecate this and the `rightScrollAriaLabel` prop and add `backScrollAriaLabel` and `forwardScrollAriaLabel`  props.

Reply:
Also updated some internal state to clean up the last bits of left/right